exercises/etl/src/main/java/Etl.java

```diff
@@ -3,6 +3,6 @@
 
 public class Etl {
     public Map<String, Integer> transform(Map<Integer, List<String>> old) {
-        return null;
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
```

Comment:
The fact that you didn't delete this starter implementation because it had a complex stub shows that you read and paid attention to the policies doc which means alot